config/settings.yml

```diff
@@ -387,8 +378,6 @@ modules_appeals_api:
       enabled: false
     weekly_decision_review:
       enabled: false
-    daily_error:
-      enabled: false
     weekly_error:
```

Comment:
I believe you can drop this key and it child for the same reasons you dropped the daily.